lib/active_yaml/base.rb

```diff
@@ -19,7 +26,9 @@ def extension
 
       private
       def load_path(path)
-        YAML.load(ERB.new(File.read(path)).result)
+        file = File.read(path)
+        result = erb_disabled ? file : ERB.new(file).result
```

Comment:
Maybe something like this?

```suggestion
        result = File.read(path)
        result = ERB.new(result).result if process_erb
```